k8s/spark-operator/apptest/tester/spark-pi.yaml.template

```diff
@@ -14,30 +14,31 @@
 # limitations under the License.
 #
 
-apiVersion: "sparkoperator.k8s.io/v1alpha1"
+apiVersion: "sparkoperator.k8s.io/v1beta1"
 kind: SparkApplication
 metadata:
-  name: $NAME
+  name: $SPARK_APP_NAME
   namespace: $NAMESPACE
```

Comment:
Not sure if we need namespace here, could you confirm that?

Reply:
I see, tester.sh applies without namespace.

IMHO It would be better to leave YAML without namespace and apply with namespace, but we do not have strict style guide on this.

Reply:
I see others like the airflow operator also sets namespace.